src/components/section.js

```diff
@@ -1,11 +1,14 @@
 import React from "react";
 import { GridContainer, Grid } from "@trussworks/react-uswds";
 
-const Section = () => {
+const Section = ({ sectionHeading, sectionText }) => {
   return (
     <GridContainer>
-      <Grid col>
-        <Grid></Grid>
+      <Grid row>
+        <Grid>
```

Comment:
Is it necessary to have another `<Grid>` here? Can we get away with using `<Grid row>` only?